README.md

```diff
@@ -27,81 +23,97 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 <!-- Below content is automatically populated via pre-commit hook -->
 <!-- BEGIN OVERVIEW HOOK -->
 ## Overview
-* [terraform-ibm-module-template](#terraform-ibm-module-template)
+* [terraform-ibm-app-configuration](#terraform-ibm-app-configuration)
 * [Examples](./examples)
     * [Basic example](./examples/basic)
     * [Complete example](./examples/complete)
 * [Contributing](#contributing)
 
-## terraform-ibm-module-template
+## terraform-ibm-app-configuration
 <!-- END OVERVIEW HOOK -->
 
 ### Usage
 
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
 ```hcl
-
+module "app_config" {
+  source                       = "terraform-ibm-modules/app-configuration/ibm"
+  version                      = "latest" # Replace "latest" with a release version to lock into a specific release
```

Comment:
Can you replace "latest" with "X.X.X" as we got feedback that this was confusing and someone thought they could just use the string "latest" to pull latest